### PR TITLE
magento/devdocs#: Custom admin URL naming

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

Fix pull request fixes https://github.com/magento/merchdocs/issues/1098

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/env/environment-vars_magento.html#admin-url

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Backend/etc/adminhtml/system.xml#L412
-  https://github.com/magento/magento2/blob/2.4-develop/lib/web/mage/validation.js#L1034

### CODE REVIEW

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @EvheniiaKolkhiieva. Thank you for your contribution. Please, check my suggestions below

File: src/cloud/env/environment-vars_magento.md [hunk 1]
```diff
@@ -32,3 +32,6 @@ If you need to change the Admin URL after installation, use the `magento-cloud v
 ```bash
 magento-cloud variable:set ADMIN_URL magento_A8v10 -e master
 ```
+
+{:.bs-callout-info}
+Use only letters (a-z or A-Z) or numbers (0-9) for a custom admin path. No spaces or other characters are allowed.
```
Comment (hguthrie) [inline src/cloud/env/environment-vars_magento.md:37]:
```suggestion
The `ADMIN_URL` value accepts letters (a-z or A-Z) or numbers (0-9) for a custom admin path. Spaces or other characters are **not** accepted.
```

File: src/cloud/env/environment-vars_magento.md [hunk 2]
```diff
@@ -32,3 +32,6 @@ If you need to change the Admin URL after installation, use the `magento-cloud v
 ```bash
 magento-cloud variable:set ADMIN_URL magento_A8v10 -e master
```
Comment (hguthrie) [inline src/cloud/env/environment-vars_magento.md:33]: If that is the rule, then the underscore here is not valid?

**Review by hguthrie — CHANGES_REQUESTED** [review]

File: src/cloud/env/environment-vars_magento.md (hunk 2 above)
Comment (atwixfirster) [reply src/cloud/env/environment-vars_magento.md:33]: Hi, @hguthrie !

We have a validation rule for the custom admin path on the **admin** side only:

magento_adm


However, a custom magento path may be set:
- in `app/etc/env.php` 
- in `ADMIN_URL` environment variable on Cloud (Cloud UI does not have any validation rules and we can type there everything we need):
cloud_adm

So, @EvheniiaKolkhiieva made good changes basic on codebase. In the same time our @rogyar is a very expirienced Magento specialist and he 100% knows that some additional chars (such as `_`) are also allowed in the custom admin path.

Thank you!

File: src/cloud/env/environment-vars_magento.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/cloud/env/environment-vars_magento.md:37]: Thank you, @rogyar, for your reviewing. 

Will be fixed in the next commit.

Comment (EvheniiaKolkhiieva) [reply src/cloud/env/environment-vars_magento.md:37]: Thank you, @hguthrie,

Will be fixed in the next commit.

File: src/cloud/env/environment-vars_magento.md (hunk 2 above)
Comment (hguthrie) [reply src/cloud/env/environment-vars_magento.md:33]: Sweet! Thanks for the detailed response!
@atwixfirster @EvheniiaKolkhiieva  And thank you both for tackling this and clearing things up. Much appreciated.

**Comment by EvheniiaKolkhiieva** [comment]
@hguthrie and @rogyar,

your suggestions was implemented. Please, check.

Thank you!

**Review by hguthrie — APPROVED** [review]

**Comment by hguthrie** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@hguthrie the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8858`.

**Pull request merged**